package/gluon-luci-node-role/files/usr/lib/lua/luci/model/cbi/admin/noderole.lua

```diff
@@ -1,23 +1,23 @@
 local f, s, o
 local site = require 'gluon.site_config'
+local i18n = require "luci.i18n"
 local uci = luci.model.uci.cursor()
 local config = 'gluon-node-info'
 
 -- where to read the configuration from
 local role = uci:get(config, uci:get_first(config, "system"), "role")
 
-f = SimpleForm("role", "Verwendungszweck")
+f = SimpleForm("role", i18n.translate("Node Role"))
 f.reset = false
 f.template = "admin/expertmode"
-f.submit = "Fertig"
+f.submit = i18n.translate("Finish")
```

Comment:
You can just leave `f.submit` unset, then it will default to "Submit" and "Absenden". I think it makes most sense to go with LuCI's defaults here.

You should also remove the line `f.reset = false` to match the other expert mode modules, which all have the reset button.